src/mame/mr/dribling.cpp

```diff
@@ -231,7 +255,6 @@ uint8_t dribling_state::input_mux0_r()
 }
 
 
-
 /*************************************
```

Comment:
You’ve unnecessarily removed a blank line here – it was using three blank lines to separate major sections.

Reply:
Done